mne/preprocessing/bads.py

```diff
@@ -15,18 +19,18 @@ def find_outliers(X, threshold=3.0):
         The scores for which to find outliers.
     threshold : float
         The value above which a feature is classified as outlier.
+    max_iter : int
+        The maximum number of iterations.
 
     Returns
     -------
     bad_idx : np.ndarray of int, shape (n_features)
         The outlier indices.
     """
-    max_iter = 2
     my_mask = np.zeros(len(X), dtype=np.bool)
-    X = np.abs(X)
     for _ in range(max_iter):
         X = np.ma.masked_array(X, my_mask)
-        this_z = stats.zscore(X)
+        this_z = np.abs(stats.zscore(X))
```

Comment:
evil ... outliers with lower than normal z-scores have been ignored. relevant for the correlation metric or low variance @wmvanvliet